tests/smoke-go.yaml

```diff
@@ -2,17 +2,17 @@ input:
     job:
         package-manager: go_modules
         allowed-updates:
-            - update-type: all
-              dependency-type: direct
+            - dependency-type: direct
+              update-type: all
+        experiments:
+            goprivate: null
```

Comment:
Why is this `null` vs previously it was empty?

Reply:
I'm not sure, but it should always be a string from the API so I will fix it up. Shouldn't matter either way, but want it to reflect reality.